### PR TITLE
Follow up PR for BergeGraphInspectorTest

### PR DESCRIPTION
**Pull request by PhilippKaesgen**
#579 As requested. Let me know if anything should be changed.

### CODE REVIEW

**Comment by jsichi** [comment]
In jgrapht-core/src/test/java/org/jgrapht, could you update FastTestSuite.java and IntegrationTestSuite.java as well?  These are used for running the suites via Eclipse or other IDE, so they should exclude OptionalTests.

File: jgrapht-core/src/test/java/org/jgrapht/FastTestSuite.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
  */
 @RunWith(ParallelSuite.class)
 @Categories.ExcludeCategory(SlowTests.class)
+@Categories.ExcludeCategory(OptionalTests.class)
```
Comment (d-michail) [inline jgrapht-core/src/test/java/org/jgrapht/FastTestSuite.java]: This does not look right! There should be only on `@Categories.ExcludeCategory` with both annotations.

Comment (PhilippKaesgen) [reply jgrapht-core/src/test/java/org/jgrapht/FastTestSuite.java]: I know, I corrected this in the last commit.

**Comment by jsichi** [comment]
Getting closer...I tested with your branch, and discovered that it's not possible to run the optional tests from the command line, even by specifying -Dtest, e.g.

mvn -Dtest=org.jgrapht.alg.cycle.BergeGraphInspectorTest#checkPappusGraph test

The command above fails since tests in excludedGroups are not visible at all.

I found a solution here:

https://stackoverflow.com/a/18734599/2913158

It's a big ugly, but it will allow us to use this command to override excludedGroups via -D:

mvn -DexcludedGroups="" -Dtest=org.jgrapht.alg.cycle.BergeGraphInspectorTest#checkPappusGraph test

In order for that to work, it's necessary to:

a) define a pom property excludedGroups with a default value of org.jgrapht.SlowTests,org.jgrapht.OptionalTests

b) in the excludedGroups tag (of which there should be only one), reference it as ${excludedGroups}

That seems to work for me, although I didn't test thoroughly.

**Comment by jsichi** [comment]
I pushed one edit to your branch.  Without it, mvn verify was no longer running the SlowTests as it should; I guess somehow the excludedGroups property was applying to failsafe implicitly.

**Comment by PhilippKaesgen** [comment]
I performed the tests as described above and it seems to work. I couldn't find a more elegant way, either. But I'm no maven expert, anyway ;-)

**Comment by PhilippKaesgen** [comment]
Ok, you're right they didn't run..

**Comment by jsichi** [comment]
Thanks, merging now!  I'll update the wiki.

**Pull request merged**